jumpscale/sals/marketplace/chatflow.py

```diff
@@ -169,9 +191,78 @@ def solution_name(self):
     @chatflow_step(title="Payment currency")
     def payment_currency(self):
         self.currency = self.single_choice(
-            "Please select the currency you want to pay with.", ["FreeTFT", "TFT", "TFTA"], required=True
+            "Please select the currency you want to pay with.", ["FreeTFT", "TFT", "TFTA"], required=True, default="TFT"
+        )
+
+    def _refund(self, transaction, effects):
+        currency = self.currency or "TFT"
+        if effects.amount < 0:
```

Comment:
shouldn't we check if amount <= 0.1 the transaction fees?

Reply:
indeed you are partially right, in TFT there's 0.1 transaction fees., TFTA&FreeTFT no but not included in the effects

so in the refund if he paid with TFT, we should deduce 0.1 from the refunded amount as a transaction fees